modules/end-user-guide/pages/navigating-openshift-web-console-from-che.adoc

```diff
@@ -0,0 +1,24 @@
+:navtitle: Navigating OpenShift web console from {prod-short}
+:keywords: overview, OpenShift, web console
+:page-aliases: .:accessing-openshift-web-console-from-che, overview:accessing-openshift-web-console-from-che, accessing-openshift-web-console-from-che
+
+:parent-context-of-accessing-che-from-openshift-developer-perspective: {context}
+
+[id="navigating-openshift-web-console-from-{prod-id-short_{context}"]
+= Navigating OpenShift web console from {prod-short}
+
+This section describes how to access OpenShift web console from {prod-short}.
```

Comment:
The OS web console.

Reply:
we do not use such contractions I believe - the official naming is OpenShift web console

Reply:
I didn't mean to contract, I meant to add an article.